i18n/en.json

```diff
@@ -4086,6 +4086,8 @@
   "shortcuts.msgs.reprint_prev.mac": "Reprint previous message:\t⌘|Up",
   "shortcuts.nav.direct_messages_menu": "Direct messages menu:\tCtrl|Shift|K",
   "shortcuts.nav.direct_messages_menu.mac": "Direct messages menu:\t⌘|Shift|K",
+  "shortcuts.nav.expand_sidebar": "Expand the sidebar:\tCtrl|Shift|.",
+  "shortcuts.nav.expand_sidebar.mac": "Expand the sidebar:\t⌘|Shift|.",
```

Comment:
Should indicate "right" as in "right sidebar".

@cwarnermm Could you please confirm which one to be used: "Expand the right sidebar" or "Expand right sidebar"? Thank you!
<img width="285" alt="Screen Shot 2022-01-06 at 3 40 58 PM" src="https://user-images.githubusercontent.com/5334504/148347383-b746e64b-a221-481d-bb54-0824db7873d7.png">

Reply:
@saturninoabril I recommend "Expand **the** right sidebar" to match the other descriptions on that UI. Thanks!

Reply:
I will do the chnages once @cwarnermm confirms